src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbChainTest.java

```diff
@@ -368,8 +365,11 @@ private void run(MtAmazonDynamoDbContextProvider mtContext,
         UpdateItemRequest updateItemRequest = new UpdateItemRequest()
             .withTableName(tableName1)
             .withKey(updateItemKey)
-            .addAttributeUpdatesEntry(SOME_FIELD,
-                new AttributeValueUpdate().withValue(createStringAttribute("someValue1Updated")));
+            .withUpdateExpression("set #someField = :someValue")
```

Comment:
This test, which is of marginal value, was @Disabled because of failures.  It happened to be using attributeUpdates so I swapped them over to use update expression fixed the expected gold file, and re-enabled.